### ISSUE TITLE
Ajax to have DiMe UI update automatically on changes

### ISSUE THREAD

**Issue by mvsjober** [HIIT/dime-server#49]

**Comment by chenhunghan** [comment]
What do you mean by automatically? Make a call (GET /api/data/events) every 10 minutes?

**Comment by mvsjober** [comment]
That's definitely not a good idea. It should probably be done with an ajax connection open at all times... I guess this is for some future version far away. I'll remove it from the milestone.

**Comment by chenhunghan** [comment]
I have done similar thing before but it's backend was using Websocket.

**Comment by mvsjober** [comment]
Yes, we would probably need WebSocket stuff for that. This is definitely a future thing... I wasn't thinking clearly when I put it to the first milestone :-)
